src/main/python/yadtminion/__init__.py

```diff
@@ -198,7 +210,18 @@ def next_artefacts_need_reboot(self):
         result = list(updates.intersection(packages_inducing_reboot))
         return result
 
+    @staticmethod
+    def is_sysv_service(service_name):
+        chkconfig = Command("/sbin/chkconfig")
+        sysv_services = [line.split("\t", 1)[0].strip() for line in chkconfig()]
```

Comment:
Why not just
```
sysv_services = [line.split()[0] for line in chkconfig()]
```
It's easier to read and works even if chkconfig ever chooses to seperate the columns with spaces instead of tabs.